Dockerfile

```diff
@@ -6,26 +6,46 @@ LABEL maintainer="Aditya Patwardhan <adityap@microsoft.com>"
 
 RUN pwsh -c Install-Module PSScriptAnalyzer -RequiredVersion 1.17.1 -Force -Confirm:\$false
 
+RUN apt-get update && apt-get --no-install-recommends -y install git=1:2.7.4-0ubuntu1.5 && \

```

Comment:
We don't want to install dependencies we will not use to run.
If you need them, `install -> use -> remove -> clean the system`

Reply:
moved dependencies to separate layer